apps/server/src/services/extra-timer-service/ExtraTimerService.ts

```diff
@@ -76,4 +76,4 @@ function broadcastReturn(_target: any, _propertyKey: string, descriptor: Propert
 const emit = (state: SimpleTimerState) => eventStore.set('timer1', state);
 const timeNow = () => Date.now();
 
-export const extraTimerService = new ExtraTimerService(emit, timeNow);
+export const extraTimerService = new Array(new ExtraTimerService(emit, timeNow));
```

Comment:
I don't think it is correct to have an array of services. But a service that manages an array of timers

With the array of services the management logic gets pushed to the consumer, defeating the point of the service